waku/common/wakubridge.nim

```diff
@@ -265,24 +265,28 @@ when isMainModule:
   if conf.logLevel != LogLevel.NONE:
     setLogLevel(conf.logLevel)
 
+  ## `udpPort` is only supplied to satisfy underlying APIs but is not
+  ## actually a supported transport.
+  let udpPort = conf.devp2pTcpPort
```

Comment:
I am not very familiar with the context of this PR, but just got curious whether this assignment would cause any issue/conflicts in the lower level protocols? it seems like a port configured for a TCP connection is also used as a `udpPort`? Or maybe the naming is misleading! :)

Reply:
@staheri14 I've tried to clarify that in the commit message + all the comments in the code where changes where made:

> The UDP port is merely used to satisfy underlying nim-eth APIs
(namely proc redirectPorts which expects a UDP port).

> Since there's no support for UDP transport in nim-waku, there's no
need to have a CLI option for that.

> This commit removes the option from the CLI while keeping the value
inside application code in tact to not break calls to third-party
APIs.

The UDP port isn't used anywhere but we need to have it unless we want to introduce optional parameters in libp2p APIs (which is a bigger effort). Hence, we've decided to just assign the TCP port for now, but it could be any value really.

Does this make sense? Happy to add more clarification to the commit if needed!

Reply:
To clarify further: 

The alternative would be to either hard code some dummy port in all the places where UDP port is used, or have a variable defined somewhere to avoid repetition.

The second option is more convenient, so we went with that use the already existing TCP port for that.